src/wxMaxima.cpp

```diff
@@ -3708,9 +3708,9 @@ void wxMaxima::SetupVariables()
   wxString cmd;
 
 #if defined (__WXOSX__)
-  wxString gnuplotbin(wxT("/Applications/Gnuplot.app/Contents/Resources/bin/gnuplot"));
-  if (wxFileExists(gnuplotbin))
-    cmd += wxT("\n:lisp-quiet (setf $gnuplot_command \"") + gnuplotbin + wxT("\")\n");
+  wxString gunplot_binary = Dirstructure::GnuplotDefaultLocation(m_gnuplotcommand);
+  if (wxFileExists(gunplot_binary))
+    cmd += wxT("\n:lisp-quiet (setf $gnuplot_command \"") + gunplot_binary + wxT("\")\n");
```

Comment:
Here I want a wxLogMessage telling the user we have changed the value of gnuplot_command: Else we or the maxima team sooner or later get a but report from a user who tries to change gnuplot_command in the startup file, which for some reason is without effect.